hr_attendance_report_theoretical_time/models/hr_employee.py

```diff
@@ -1,15 +1,25 @@
 # Copyright 2018 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import fields, models, api
 
 
 class HrEmployee(models.Model):
     _inherit = 'hr.employee'
 
+    @api.multi
+    def _compute_theoretical_hours_difference(self):
+        data = self.env['hr.attendance.theoretical.time.report'].read_group([('employee_id', 'in', self.ids)],
```

Comment:
This doesn't conform flake8 80 col limit

Reply:
fixed